README.md

```diff
@@ -84,6 +84,7 @@ Make sure to take a look at the [documentation](https://simulatedgreg.gitbooks.i
 ## Made with electron-vue
 Take a look at some of the amazing projects built with electron-vue. Want to have your own project listed? Feel free to submit a pull request.
 
+* [**GitHoard**](https://github.com/jojobyte/githoard): Hoard git repositories with ease
```

Comment:
Please pull changes, and move to bottom.

Reply:
Done!